python/GafferArnold/ArnoldRender.py

```diff
@@ -64,6 +64,13 @@ def __init__( self, name="ArnoldRender" ) :
 		)
 
 		self.addChild(
+			Gaffer.StringPlug(
+				"command",
+				defaultValue = "kick -dp -dw",
```

Comment:
It just seemed odd to expose the command but not expose the args that aren't otherwise controllable via plugs.